### DISCUSSION TITLE
Preview for translation

### DISCUSSION THREAD

**Discussion by do-web** [directus/directus#20531] [Feature Requests]
### Summary

Currently a preview for translation is not possible.

### Motivation

Currently a preview for translation is not possible.

### Detailed Design

Each translation should have a preview URL.

### Requirements List

Each translation should have a preview URL.

### Drawbacks

Each translation should have a preview URL.

### Alternatives

Each translation should have a preview URL.

### Adoption Strategy

Each translation should have a preview URL.

**Comment by rijkvanzanten** [comment]
Closing this as the RFC isn't filled out
